.eslintrc.js

```diff
@@ -40,8 +40,47 @@ module.exports = {
     {
       'files': ['playground/**/*.json'],
       'rules': {
-        'local-rules/no-missing-api-keys': 1,
-      }
+        'local-rules/no-missing-api-keys': 2,
+      },
+      'overrides': [
+        {
+          'files': [
+            'playground/**/authenticator-expiry-warning-password.json',
```

Comment:
These mocks have known issues. Will create tickets for home teams to address.

Reply:
This is a good way to make sure the rule gets enforced by default for all **new mock files**. 🎉 

Am curious if some of these error files may actually be **correct**, as the error message is localized before the Widget consumes it. If so - are we encouraging developers to make backend changes to add an i18n property or do we intend to have another category if "ignored" files?

Reply:
We want to make sure all API response have an i18n key. These are mocks that either have a i18n key itself missing from the mock or they have it in the mock but is not added to the login bundle. For cases where its missing from the mock, we want an API change to be made by the home team.